app/views/registers/show.html.haml

```diff
@@ -2,142 +2,139 @@
+                        %span.visually-hidden
+                          What does #{field['field']} mean
+                        ?
+              %tbody#records-tbody
+                - if @register.is_empty?
+                  %tr
+                    %td{colspan: @register.register_fields.count}
+                      .panel.panel-border-wide
+                        %p This register currently has no data. The owner of the register will publish the data once it has been gathered.
+                - else
+                  = render partial: 'record', collection: @records
+          .pager
+            .pager-controls.js-hidden
+              = paginate @records, outer_window: 0, window: 3
+            - unless @records.last_page?
+              %div
+                = link_to_next_page @records, 'Load more', id: 'load-more-records', class: 'js-load-more', remote: true
 
-              - unless @records.any? || @register.is_empty?
-                .no-search-results
-                  %p
-                    - if @search_term.present?
-                      No results found for <strong>"#{@search_term}"</strong>
-                      %script
-                        var searchFilter = document.getElementById('search_records').querySelector('input[type="radio"]:checked').parentElement.querySelector('label').innerText.toLowerCase();
-                        ga('send', 'event', 'Search', 'No results filtered by: ' + searchFilter, '#{@search_term}', {nonInteraction: true});
-                    - else
-                      No results found.
-                      %script
-                        ga('send', 'event', 'Search', 'No results', 'Search term not present');
-                    - if @search_term.present? || params[:status].present?
-                      = link_to 'Reset', register_path(@register.slug, anchor: 'search_wrapper'), class: 'reset-link'
-          - if @records.any?
-            .subsection--not-accordionised
-              .subsection__header--not-accordionised
-                %h2.subsection__title--not-accordionised
-                  = link_to 'Access the data', register_choose_access_path(@register.slug)
-          .subsection.js-subsection
-            .subsection__header
-              %h2.subsection__title Give feedback
-            .subsection__content.js-subsection-content#feedback_section
-              = render partial: 'feedback/form'
+      .govuk-grid-column-full.subsection__content#about-the-data__5
+        %p= link_to 'Access all the data in this register', register_choose_access_path(@register.slug), { id: 'access-the-data-from-preview', class: 'registers__button--access-large'}
+        - unless @records.any? || @register.is_empty?
+          - if @search_term.present?
+            %script
+              -# TODO: current and archived no longer present; what goes here as the filter?
+              var searchFilter = 'current'; //
```

Comment:
If we are removing the filtering options I guess we should remove this extra tracking logic too?

Reply:
Fixed.